offlineimap.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Startup from single-user installation
 # Copyright (C) 2002 - 2008 John Goerzen
 # <jgoerzen@complete.org>
```

Comment:
This commit will be discarded by a rebase on top of the current public next branch. ,-)

Reply:
OK, no problem!

Reply:
Leaving in this commit, so that the Travis tests on `python3` will run and pass.  As soon as the encrypted login vars are added.  Without having to wait for the current public next branch.

Reply:
These changes are already in the current 'next' branch. I was expecting you to rebase on the current 'next' branch. :-)
I you do so, these hunks will disappear.

Reply:
Anyway, I'll apply.